spec/defines/pool_spec.rb

```diff
@@ -39,6 +39,27 @@
     }
   end
 
+  describe 'with range array' do
+    let :params do {
+      :network => '10.0.0.0',
+      :mask    => '255.255.255.0',
+      :range   => ['10.0.0.10 - 10.0.0.50','10.0.0.100 - 10.0.0.150'],
+    } end
+
+    it {
+      verify_concat_fragment_exact_contents(catalogue, 'dhcp.conf+70_mypool.dhcp', [
+        "subnet 10.0.0.0 netmask 255.255.255.0 {",
```

Comment:
oh, I see... we probably need to establish a styleguide and rubocup checks or similar also here and then change everything whatever we agree on :)